config/standard/translations/messages-bm.properties

```diff
@@ -924,225 +924,225 @@ contact.profile.imm.cholera = kunfilanintu boloci
+messages.schedule.child.month_10 =
+messages.schedule.child.month_11 =
+messages.schedule.child.month_12 =
+messages.schedule.child.month_13 =
+messages.schedule.child.month_14 =
+messages.schedule.child.month_15 =
+messages.schedule.child.month_16 =
+messages.schedule.child.month_17 =
+messages.schedule.child.month_18 =
+messages.schedule.child.month_19 =
+messages.schedule.child.month_20 =
+messages.schedule.child.month_21 =
+messages.schedule.child.month_22 =
+messages.schedule.child.month_23 =
+messages.schedule.child.month_24 =
+messages.schedule.child.month_25 =
+messages.schedule.child.month_26 =
+messages.schedule.child.month_27 =
+messages.schedule.child.month_28 =
+messages.schedule.child.month_29 =
+messages.schedule.child.month_30 =
+messages.schedule.child.month_31 =
+messages.schedule.child.month_32 =
+messages.schedule.child.month_33 =
+messages.schedule.child.month_34 =
+messages.schedule.child.month_35 =
+messages.schedule.child.month_36 =
+messages.schedule.child.month_37 =
+messages.schedule.child.month_38 =
+messages.schedule.child.month_39 =
+messages.schedule.child.month_40 =
+messages.schedule.child.month_41 =
+messages.schedule.child.month_42 =
+messages.schedule.child.month_43 =
+messages.schedule.child.month_44 =
+messages.schedule.child.month_45 =
+messages.schedule.child.month_46 =
+messages.schedule.child.month_47 =
+messages.schedule.child.month_48 =
+messages.schedule.child.month_49 =
+messages.schedule.child.month_50 =
+messages.schedule.child.month_51 =
+messages.schedule.child.month_52 =
+messages.schedule.child.month_53 =
+messages.schedule.child.month_54 =
+messages.schedule.child.month_55 =
+messages.schedule.child.month_56 =
+messages.schedule.child.month_57 =
+messages.schedule.child.month_58 =
+messages.schedule.child.month_59 =
+messages.schedule.child.month_60 =
+report.child_health_registration.patient_id =
+report.child_health_registration.patient_name =
+report.child_health_registration.chw_name =
+report.child_health_registration.chw_phone =
+report.child_health_registration.birth_date =
+report.child_health_registration.chw_sms =
+contact.last.visited.date =
+contact.last.visited.unknown =
+contacts.results.sort =
+contacts.results.sort.title =
+contacts.results.sort.alpha =
+contacts.results.sort.date.last.visited =
+status.review.title =
+status.review.unverified =
+status.review.errors =
+status.review.correct =
+status.sms.title =
+status.sms.valid =
+status.sms.invalid =
+report.delivery.delivery_code_label =
+report.delivery.pregnancy_outcome_label =
+task.pregnancy_danger_sign.title =
+task.pregnancy_missing_birth.title =
+task.pregnancy_missing_visit.title =
+task.postnatal_home_birth.title =
+task.postnatal_danger_sign.title =
+task.postnatal_missing_visit.title =
+task.immunization_missing_visit.title =
+task.warning.danger_sign =
+task.warning.high_risk =
+task.warning.home_birth =
+mrdt.disabled =
+mrdt.verify =
+contacts.visits.visits =
+contacts.visits.count =
```

Comment:
Unfortunately updating translations like this doesn't work. Have a read of [our documentation](https://github.com/medic/medic-docs/blob/master/development/translations.md). Essentially any changes will be overwritten next time we export from POE.

If any of the values have changed we need to change them in POE.

The new translations which you've correctly included in messages-end.properties will need to be imported in to POE after merge and then translated to the various languages and exported back into the repo.